internal/pkg/cli/svc_deploy.go

```diff
@@ -262,6 +263,14 @@ func (o *deploySvcOpts) configureClients() error {
 	// CF client against env account profile AND target environment region
 	o.svcCFN = cloudformation.New(envSession)
 
+	o.endpointGetter, err = describe.NewEnvDescriber(describe.NewEnvDescriberConfig{
```

Comment:
Do we need to update `cli/deploy.go` as well? I couldn't find it in the list of files.

Reply:
No, since there are no flag changes and `deploy` under the hood just invokes svc or job deploy.

Reply:
The `endpointGetter` func is initialized during `Execute` for both `svc init` and `job init` by the `configureClients()` method on the `initSvcProps` and `initJobProps` structs.